react/daily-art-prompt/src/components/ImageLayout.js

```diff
@@ -4,37 +4,58 @@ import Image from './Image.js'
 import CommentLayout from './CommentLayout'
 import * as TYPES from '../store/actions'
 
-export const ImageLayout = ({ onDoubleClick, image, onKeyDown, deleteComment }) =>
-  <div className="column">
-    <Image
-      image={image}
-      onDoubleClick={() => onDoubleClick(image.id)}
-    />
-    <div
-      data-testid="likedDiv" className={image.liked ? "likedText" : "hidden"}>
-      Liked
+export const ImageLayout = ({
+  onDoubleClick,
+  image,
+  onKeyDown,
+  deleteComment,
+  cancelCommentClassNames,
+  editComment,
+  editingCommentClassNames
+}) => {
+
+  const handleSubmit = (e, imageId, commentId) => {
+    if (e.keyCode === 13) {
+      const value = e.target.value
+      editComment(imageId, commentId, value)
+    }
+  }
+  return (
+    <div className="column">
+      <Image
+        image={image}
+        onDoubleClick={() => onDoubleClick(image.id)}
+      />
+      <div
+        data-testid="likedDiv" className={image.liked ? "likedText" : "hidden"}>
+        Liked
     </div>
-    <div>
       <div>
-        {image.comments && image.comments.map(comment =>
-          !comment.deleted && (
-            <CommentLayout
-              data-testid={comment.id}
-              key={`comment-${comment.id}-${image.id}`}
-              comment={comment}
-              onClick={() => deleteComment(image.id, comment.id)}
-            />
-          )
-        )}
+        <div>
+          {image.comments && image.comments.map(comment =>
+            !comment.deleted && (
+              <CommentLayout
+                data-testid={comment.id}
```

Comment:
You should probably make your data-testid something more like `comment-${comment.id}`